### PR TITLE
[APIView]  Fix Summary comment no-longer appearing for Java/Swift 

### PR DESCRIPTION
**Pull request by AlitzelMendez**
## Local Validation 
image

## Issue
https://github.com/Azure/azure-sdk-tools/issues/11211

## Work
- Fixed the error by assigning the `FIRST_ROW` elementId to the summary comment
- Added tests in both ends, where the `FIRST_ROW` is added and when is readed

### CODE REVIEW

**Review by tjprescott — COMMENTED** [review]
This LGTM. My only question: if a token file _has_ an elementId on the first line (like Python), does this logic overwrite it, or is it just a default fallback?  Might but worth adding a test case for when line 1 has an element ID (and maybe you already do)

**Comment by AlitzelMendez** [comment]
> This LGTM. My only question: if a token file _has_ an elementId on the first line (like Python), does this logic overwrite it, or is it just a default fallback? Might but worth adding a test case for when line 1 has an element ID (and maybe you already do)

that is a great point!!!!!!

File: src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                                     CreatedOn = DateTime.UtcNow,
                                     ReviewId = jobInfo.APIRevision.ReviewId,
                                     APIRevisionId = jobInfo.APIRevision.Id,
-                                    ElementId = comment.Source == SummarySource ? CodeFileHelpers.FirstRowElementId : codeLine.lineId,
+                                    ElementId = codeLine.lineId ?? (comment.Source == SummarySource ? CodeFileHelpers.FirstRowElementId : null),
```
Comment (tjprescott) [inline src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:100]: Sweet. Let's add a test case to make sure this special casing behavior is protected

Comment (AlitzelMendez) [reply src/dotnet/APIView/APIViewWeb/HostedServices/CopilotPollingBackgroundHostedService.cs:100]: sure! I added couple of more test to include the scenario in where is null but not summary, summary not null, summary null and standard not summary not null :)
